src/uct/ib/base/ib_iface.c

```diff
@@ -492,6 +495,19 @@ static ucs_status_t uct_ib_iface_create_cq(uct_ib_iface_t *iface, int cq_length,
     }
 out:
     return status;
+#else
```

Comment:
maybe the `#else` should be in line 475? to avoid code duplication of calling ibv_create_cq()

Reply:
I guess you mean #endif on line 475? It's problematic to pass coverity then - unsetenv becomes deadcode.

Reply:
we can put "env_var_added" under #ifdef.. would still be shorter